modules/op-installing-tkn-on-linux-using-rpm.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/tkn_cli/installing-tkn.adoc
+
+[id="installing-tkn-on-linux-using-rpm"]
+
+= Installing {pipelines-title} CLI (tkn) on Linux using an RPM
+
+For {op-system-base-full} version 8, you can install the {pipelines-title} CLI (`tkn`) as an RPM if you have an active {product-title} subscription on your Red Hat account.
+
+.Prerequisites
+
+* Must have root or sudo privileges.
+
+.Procedure
+
+. Register with Red Hat Subscription Manager:
++
+----
+# subscription-manager register
+----
+
+. Pull the latest subscription data:
++
+----
+# subscription-manager refresh
+----
+
+. List the available subscriptions:
++
+----
+# subscription-manager list --available --matches '*pipelines*'
+----
+
+. In the output for the previous command, find the pool ID for an {product-title} subscription and attach the subscription to the registered system:
++
+----
+# subscription-manager attach --pool=<pool_id>
+----
+
+. Enable the repositories required by {pipelines-title}:
++
+----
+# subscription-manager repos --enable="pipelines-1.0-for-rhel-8-x86_64-rpms"
+----
+
+. Install the `openshift-pipelines-client` package:
++
+----
+# yum install openshift-pipelines-client
+----
+
+After you install the CLI, it is available using the `tkn` command:
```

Comment:
Suggestion: add an explicit verification step like `tkn version`

Reply:
good idea! Added the command